client/utils/tags.js

```diff
@@ -1,12 +1,16 @@
 import React from 'react';
 import { withPrefix } from 'gatsby';
 import i18next from 'i18next';
+import psl from 'psl';
+import { homeLocation } from '../../config/env.json';
```

Comment:
```suggestion
import env from '../../config/env.json';

const { homeLocation } = env;
```
I'm not 100% sure how this interacts with Webpack (but they've [dropped support for this](https://webpack.js.org/blog/2020-10-10-webpack-5-release/#json-modules)), so let's err on the side of caution.